### ISSUE TITLE
IIIF

### ISSUE THREAD

**Issue by alnopa9** [alnopa9/Banksy#33]
Banksy Team,
Hello! So, I just finished up a workshop on IIIF (read about it) and think it would be perfect to implement into our project! There's a lot to do for, but I think the pay off would be great!

**Comment by alnopa9** [comment]
Things to do:
- [ ] Convert each image file (.img and .png) to [IIIF API standards](https://iiif.io/api/image/2.1/#table-of-contents)
    * [An example .json file](https://iiif.io/api/image/2.1/#complete-response)
- [ ] Upload images to newtfire server
- [ ] Create different galleries (one for each gallery page; graffiti, prints, exhibits, and miscellaneous)
- Update CSS to reflect new IIIF changes

**Comment by ebeshero** [comment]
@alnopa9 : Let's experiment with Cantaloupe and other IIIF servers on newtfire, but you will want to experiment locally first to get a sense of how these image servers work! See https://iiif.io/apps-demos/#image-viewing-clients

**Comment by alnopa9** [comment]
@ebeshero : Yes, I need to find another IIIF server to use on my computer because Cantaloupe just does not want to work.

**Comment by alnopa9** [comment]
**NEWS**
Some of Banksy's books are already IIIF compatible! 🎉 🎉 🎉 
* [Existencilism](https://iiif.archivelab.org/iiif/fp_Banksy-Existencilism)
* [Wall & Piece](https://iiif.archivelab.org/iiif/fp_Banksy-Wall_And_Piece)
* [Banging Your Head Against A Brick Wall](https://iiif.archivelab.org/iiif/fp_Banksy-Banging_Your_Head_Against_A_Brick_Wall)
* Cut It Out is on archive.org, but you have to log in to be able to 'borrow' the .json file. Because of this, I'm not sure if we would be able to share that publicly.
You can view these in a IIIF viewer (just click the link)

This means we also need to determine how and where we will be adding these books to the site. I think we should place a link for each book on its individual gallery page. Maybe put it in a link that says 'Read Me Here' and that link can take you to the viewer?

I'd also like to add annotations to the images in the books that include the title of the work and whether we have the work on our site or not.

**Comment by alnopa9** [comment]
**Update**
We discussed about where to host our Banksy images as IIIF and decided to not consider the newtfire server. I started to upload our images onto archive.org, but we need to make sure that we have high quality images to use with IIIF and to be sure to credit our sources properly.
